man/git-feature.md

```diff
@@ -106,8 +107,8 @@ You can configure the default branch prefix and separator via git config options
 
   $ git feature -- finish remote  
   ...  
-  $ (feature/finish-remote) git commit -m "Some changes"
-  $ (feature/finish-remote) git checkout main
+  $ (feature/finish-remote) git commit -m "Some changes"  
```

Comment:
Would you remove the trailing space here? Thanks!

Reply:
That trailing space is necessary to make the next line actually break. It matches the other lines elsewhere in this and other docs.

See https://github.com/tj/git-extras/pull/1123#discussion_r1424895964 for the problem that this solved.

https://www.markdownguide.org/basic-syntax/#line-breaks

Personally I really dislike that syntax - invisible characters that change the meaning of something (like EOL spaces or tabs) should be outlawed ;)

Reply:
@joshka 
I see. Thanks for your explanation!